src/coverity_model.c

```diff
@@ -0,0 +1,83 @@
+typedef void (*NoitHashFreeFunc)(void *);
+typedef int (*eventer_func_t)
+             (struct _event *e, int mask, void *closure, struct timeval *tv);
+
+int mtev_conf_get_string(void *base, char *path, char **val) {
+  int is_ok;
+  if(is_ok) {
```

Comment:
What is this doing? It looks like it's declaring an uninitialized variable, then immediately checking the value.

Reply:
It's not C. it's just coverity modeling stuff.  It doesn't care what is_ok is... it is both true and false and the function is evaluated in the context of the return...  "what would happen in here if the function ultimately returns 1?"